From edd6588a82dbde6784dd8a52bd0c34addbb5e798 Mon Sep 17 00:00:00 2001
From: John Zhuge <jzhuge@cloudera.com>
Date: Wed, 17 May 2017 20:12:52 -0700
Subject: [PATCH 2443/2748] HADOOP-14196. Azure Data Lake doc is missing
 required config entry. Contributed by Atul
 Sikaria

(cherry picked from commit 03955612b77f250d53b8f7e4268338e5b78a3e66)

Change-Id: I19b43a8d99c20f63a81f7835a60ed4355754ee86
---
 .../src/site/markdown/index.md                     |    5 +++++
 1 file changed, 5 insertions(+)

diff --git a/hadoop-tools/hadoop-azure-datalake/src/site/markdown/index.md b/hadoop-tools/hadoop-azure-datalake/src/site/markdown/index.md
index 3a16253..3c4c70e 100644
--- a/hadoop-tools/hadoop-azure-datalake/src/site/markdown/index.md
+++ b/hadoop-tools/hadoop-azure-datalake/src/site/markdown/index.md
@@ -133,6 +133,11 @@ Add the following properties to your `core-site.xml`
 
 ```xml
 <property>
+  <name>dfs.adls.oauth2.access.token.provider.type</name>
+  <value>ClientCredential</value>
+</property>
+
+<property>
   <name>dfs.adls.oauth2.refresh.url</name>
   <value>TOKEN ENDPOINT FROM STEP 7 ABOVE</value>
 </property>
-- 
1.7.9.5

